modules/ROOT/pages/api-groups-promote-api-instance.adoc

```diff
@@ -0,0 +1,26 @@
+= Promote API Instances in API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can promote an API Group instance from one environment to another, for example, Sandbox to Production.
+
+Promoting an API Group instance affects only the API definition, not the existing deployed applications subscribed to the API Group. After promoting an API Group instance, the API Name remains the same, but the API Version changes.
```

Comment:
I don't know where did you get this, but I think you are describing the promotion for an API instance. Promoting a group instance does not change anything in existing group instances, the only it does is create a group instance from a specified group instance in another environment. The API instances of the promoted group instance may be different than the ones in the original group instance, but the definition (the related assets) remains the same.

Reply:
Done! Thanks!